helm/cluster-operator/templates/configmap.yaml

```
@@ -46,3 +46,4 @@ data:
           config:
             default: {{ toYaml .Values.release.app.config.default | indent 12 }}
             override: {{ toYaml .Values.release.app.config.override | indent 12 }}
+    kiamWatchdogEnabled: '{{ .Values.kiamWatchdogEnabled }}'
```

```suggestion
            kiamWatchdogEnabled: '{{ .Values.kiamWatchdogEnabled }}'
            override: {{ toYaml .Values.release.app.config.override | indent 12 }}
```

Could you move the flag so its under `service.release.app.config`?

Also the flag needs to be added [here](https://github.com/giantswarm/cluster-operator/blob/649ca383313a75494b1dba09986a50aef171d509/flag/service/release/app/config/config.go#L5) and [main.go](https://github.com/giantswarm/cluster-operator/blob/bef41996a094eca356733cbe8f2cfd40a2efa4f0/main.go#L129).